.swiftlint.yml

```diff
@@ -29,7 +29,6 @@ opt_in_rules:
   - identical_operands
   - implicit_return
   - legacy_random
-  - lower_acl_than_parent
```

Comment:
Following the Point-Free approach, I made `ViewModifier` structs `private`. This change caused this SwiftLint rule to generate many warnings. Because I found these warnings unhelpful, I removed this rule.